velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.cpp

```diff
@@ -96,17 +88,88 @@ std::unique_ptr<WriteFile> HdfsFileSystem::openFileForWrite(
   return std::make_unique<HdfsWriteFile>(impl_->hdfsClient(), path);
 }
 
-bool HdfsFileSystem::isHdfsFile(const std::string_view filename) {
-  return filename.find(kScheme) == 0;
+bool HdfsFileSystem::isHdfsFile(const std::string_view filePath) {
+  return filePath.find(kScheme) == 0;
+}
+
+/**
```

Comment:
This style of multi-line comments is not preferred. See https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md
Use `///`

Reply:
Just aware of this code style issue. Fixed in the new commit. Thanks!